app/frontend/components/shared/help-drawer.tsx

```diff
@@ -44,22 +57,85 @@ export function HelpDrawer({ defaultButtonProps, renderTriggerButton }: IProps)
         <DrawerContent display={"flex"} flexDir={"column"} h={"full"} maxW={"430px"}>
           <DrawerCloseButton fontSize={"xs"} />
           <DrawerHeader display={"flex"} alignItems={"center"} p={6} pt={10} fontSize={"2xl"}>
-            <Info style={{ marginRight: "var(--chakra-sizes-2)" }} />
             {t("ui.help")}
           </DrawerHeader>
 
-          <DrawerBody
-            sx={{
-              iframe: {
-                w: "full",
-                h: "full",
-              },
-            }}
-          >
-            <iframe src="https://www2.gov.bc.ca/gov/content?id=A5A88A4CE1D54D95AB23D57858EF11EE" title="Help"></iframe>
+          <DrawerBody>
+            {helpLinkItems && (
+              <Flex direction="column" gap={6}>
+                {Object.keys(helpLinkItems)
+                  .filter((key) => helpLinkItems[key].show)
```

Comment:
might better to move this logic to a `view` in the MST model as that should be a bit more performant